### PR TITLE
Adds backgrounds to character setup.

### PR DESCRIPTION
**Pull request by Belsima**
Port from https://github.com/VOREStation/VOREStation/pull/2341

image

### CODE REVIEW

**Comment by Iamgoofball** [comment]
i thought you guys didn't have the doggebeest

**Comment by Belsima** [comment]
We don't. Gif was borred from VORE's devs because I Don't Know How To Gif Without Choppy.

File: code/modules/mob/new_player/preferences_setup.dm [hunk 1]
```diff
@@ -257,8 +257,9 @@
 	var/mob/living/carbon/human/dummy/mannequin/mannequin = get_mannequin(client_ckey)
 	mannequin.delete_inventory(TRUE)
 	dress_preview_mob(mannequin)
+	var/icon/bgstate = pick("000", "FFF", "steel", "white", "plating", "reinforced")
 
-	preview_icon = icon('icons/effects/effects.dmi', "nothing")
+	preview_icon = icon('icons/effects/128x48.dmi', bgstate")
```
Comment (Atermonera) [inline code/modules/mob/new_player/preferences_setup.dm]: Missing "

**Comment by Anewbe** [comment]
In honesty, this looks incredibly annoying, if all it does is cycle through floors. If it was possible to choose the background, including the default black, I'd not have this complaint.

**Comment by Belsima** [comment]
Alright. I'll look into that much, then.

**Comment by Belsima** [comment]
Closing this for now, given the PR for nicknames would cause conflicts; I've also a few ideas on how to better implement this, that I'll be experimenting with, so this probably won't be finished for a bit.

**Comment by mustafakalash** [comment]
Baystation12/Baystation12#19144 has implemented a button to select and save a background choice.

**Comment by Anewbe** [comment]
Thank you.

**Pull request closed without merging**